src/main/java/org/jenkinsci/plugins/p4/client/ClientHelper.java

```diff
@@ -130,7 +130,6 @@ public boolean updateFiles() throws Exception {
 
 		// Sync revision to re-edit
 		SyncOptions syncOpts = new SyncOptions();
-		syncOpts.setNoUpdate(true);
```

Comment:
Why did you remove the sync '-n' flag?  This is used to run a preview sync to see if the workspace is up-to-date.